symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue

```diff
@@ -0,0 +1,374 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h6">General Information</oxd-text>
+        <oxd-switch-input
+          v-model="editable"
+          optionLabel="Edit"
+          labelPosition="left"
+        />
+      </div>
+      <oxd-divider />
+
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <oxd-form-row>
+              <oxd-input-field
+                label="Organization Name"
+                v-model="organization.name"
+                :rules="rules.name"
+                :disabled="!editable"
+                required
+              />
+            </oxd-form-row>
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <oxd-form-row>
+              <oxd-input-field
+                label="Number of Employees"
+                v-model="organization.noOfEmployees"
+                :rules="rules.noOfEmployees"
```

Comment:
Is the use of a disabled input field approved for this?
also if using this, no need for `:rules`

Reply:
yes, removed!